db/models.py

```diff
@@ -50,3 +53,60 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(on_delete=CASCADE, to="User")
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+    class Meta:
+        ordering = ("-created_at",)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(to=MovieSession, on_delete=CASCADE)
+    order = models.ForeignKey(to=Order, on_delete=CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=[
+                    "movie_session",
+                    "row",
+                    "seat"
+                ], name="unique_seat_per_session"
+            )
+        ]
+
+    def clean(self) -> None:
+        if self.row < 1 or self.row > 10:
+            raise ValidationError({
+                "row":
+                    "row number must be in available range: (1, rows): (1, 10)"
+            })
+        if self.seat < 1 or self.seat > 12:
+            raise ValidationError({
+                "seat": (
+                    "seat number must be in available range: "
+                    "(1, seats_in_row): (1, 12)"
+                )
+            })
+
+    def save(self, *args, **kwargs) -> None:
+        self.full_clean()
+        super().save(*args, **kwargs)
+
+    def __str__(self) -> str:
+        return (
+            f"{self.movie_session.movie.title} {self.movie_session.show_time} "
+            f"(row: {self.row}, seat: {self.seat})"
+        )
+
+
+class User(AbstractUser):
+    pass
```

Comment:
The `User` model is not extended. While this is not necessarily an error, it might limit the flexibility of your application. Consider adding extra fields that might be needed in your application, like `date_of_birth`, `profile_picture`, etc.